Fix error GL07

alexander-ponomaroff opened the pull request:
- [ ] refs #24125 
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

Original number of errors: 286
Remaining errors: 24

alexander-ponomaroff committed:
Fix error GL07

alexander-ponomaroff commented:
**ORIGINAL LIST OF ERRORS**

pandas.ExcelWriter: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, Notes, Examples
pandas.read_json: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.io.json.build_table_schema: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes, Examples
pandas.read_html: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.read_sql_table: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.read_sql_query: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.pivot_table: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.crosstab: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes, Examples
pandas.merge: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.merge_ordered: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.merge_asof: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.concat: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.get_dummies: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.unique: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.wide_to_long: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes, Examples
pandas.to_numeric: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.to_datetime: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.bdate_range: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes, Examples
pandas.period_range: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes, Examples
pandas.interval_range: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.eval: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes
pandas.Series.astype: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.infer_objects: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.Series.convert_objects: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also
pandas.Series.at: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.Series.iat: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.Series.loc: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.Series.add: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.sub: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.mul: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.div: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.truediv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.floordiv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.mod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.pow: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.radd: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rsub: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rmul: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rdiv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rtruediv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rfloordiv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rmod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rpow: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.lt: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.gt: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.le: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.ge: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.ne: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.eq: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.rolling: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.expanding: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.ewm: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.pipe: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.Series.abs: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Series.between: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.clip: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.cummax: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.cummin: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.cumprod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.cumsum: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.nlargest: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.nsmallest: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.nonzero: Sections are in the wrong order. Correct order is: See Also, Examples
pandas.Series.drop: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.duplicated: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.first: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.idxmax: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes, Examples
pandas.Series.idxmin: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes, Examples
pandas.Series.last: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.reindex: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.where: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.mask: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.filter: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.fillna: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.argmin: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes, Examples
pandas.Series.argmax: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes, Examples
pandas.Series.unstack: Sections are in the wrong order. Correct order is: Parameters, Returns, Examples
pandas.Series.append: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.asfreq: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.asof: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.resample: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.at_time: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.between_time: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.tshift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.Series.slice_shift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.Series.dt.dayofweek: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.Series.dt.weekday: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.Series.dt.to_period: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.dt.to_pytimedelta: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.Series.str.count: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.str.join: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.str.get_dummies: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.api.types.CategoricalDtype: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also, Notes, Examples
pandas.Categorical: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, Raises, See Also, Notes, Examples
pandas.Series.cat.rename_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Series.cat.reorder_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.Series.cat.add_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.Series.cat.remove_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.Series.cat.set_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.Series.to_excel: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.Series.to_xarray: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.DataFrame: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.DataFrame.select_dtypes: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, Notes, Examples
pandas.DataFrame.values: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.DataFrame.empty: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.DataFrame.astype: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.convert_objects: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also
pandas.DataFrame.infer_objects: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.DataFrame.at: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.DataFrame.iat: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.DataFrame.loc: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.DataFrame.iterrows: Sections are in the wrong order. Correct order is: Yields, See Also, Notes
pandas.DataFrame.itertuples: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.DataFrame.where: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.mask: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.query: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.add: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.sub: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.mul: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.div: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.truediv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.floordiv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.mod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.pow: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.radd: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rsub: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rmul: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rdiv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rtruediv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rfloordiv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rmod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rpow: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.combine: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.combine_first: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.apply: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.pipe: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.DataFrame.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.rolling: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.expanding: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.ewm: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.abs: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.DataFrame.clip: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.cummax: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.cummin: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.cumprod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.cumsum: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.at_time: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.DataFrame.between_time: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.DataFrame.drop: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.DataFrame.filter: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.first: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.DataFrame.idxmax: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes
pandas.DataFrame.idxmin: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes
pandas.DataFrame.last: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.DataFrame.reindex: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.reindex_axis: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.fillna: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.pivot_table: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.unstack: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.to_xarray: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.DataFrame.append: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.join: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.merge: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.asfreq: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.asof: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.slice_shift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.DataFrame.tshift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.DataFrame.resample: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.from_csv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also
pandas.DataFrame.to_excel: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.Panel.values: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Panel.astype: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.at: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.Panel.iat: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.Panel.loc: Sections are in the wrong order. Correct order is: Raises, See Also, Examples
pandas.Panel.apply: Sections are in the wrong order. Correct order is: Parameters, Returns, Examples
pandas.Panel.abs: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Panel.clip: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.cummax: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.cummin: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.cumprod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.cumsum: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.filter: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Panel.first: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Panel.last: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Panel.reindex: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.reindex_axis: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.transpose: Sections are in the wrong order. Correct order is: Parameters, Returns, Examples
pandas.Panel.asfreq: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Panel.resample: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Index.memory_usage: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.Index.duplicated: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Index.shift: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Index.slice_locs: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.RangeIndex: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also
pandas.Int64Index: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also, Notes
pandas.UInt64Index: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also, Notes
pandas.Float64Index: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also, Notes
pandas.CategoricalIndex.rename_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.CategoricalIndex.reorder_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.CategoricalIndex.add_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.CategoricalIndex.remove_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.CategoricalIndex.set_categories: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also
pandas.IntervalIndex: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also, Examples
pandas.IntervalIndex.from_arrays: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes, Examples
pandas.IntervalIndex.from_tuples: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.IntervalIndex.from_breaks: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.IntervalIndex.is_overlapping: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.IntervalIndex.overlaps: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.MultiIndex: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, See Also, Notes
pandas.MultiIndex.from_arrays: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.MultiIndex.from_tuples: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.MultiIndex.from_product: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DatetimeIndex.dayofweek: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.DatetimeIndex.weekday: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.DatetimeIndex.to_period: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.TimedeltaIndex: Sections are in the wrong order. Correct order is: Parameters, Attributes, Methods, Notes
pandas.Timestamp.round: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises
pandas.Interval: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.Interval.overlaps: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Timedelta.isoformat: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Timedelta.round: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises
pandas.core.window.Rolling.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.window.Rolling.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.core.window.Expanding.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.window.Expanding.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.core.groupby.GroupBy.apply: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.groupby.GroupBy.pipe: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.groupby.GroupBy.cumcount: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.core.groupby.GroupBy.head: Sections are in the wrong order. Correct order is: See Also, Examples
pandas.core.groupby.GroupBy.mean: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.core.groupby.GroupBy.ngroup: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.core.groupby.GroupBy.nth: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.core.groupby.GroupBy.tail: Sections are in the wrong order. Correct order is: See Also, Examples
pandas.core.groupby.DataFrameGroupBy.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.groupby.DataFrameGroupBy.fillna: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.core.groupby.DataFrameGroupBy.filter: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes, Examples
pandas.core.groupby.DataFrameGroupBy.idxmax: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes
pandas.core.groupby.DataFrameGroupBy.idxmin: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Notes
pandas.core.groupby.DataFrameGroupBy.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.core.groupby.DataFrameGroupBy.tshift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.core.groupby.SeriesGroupBy.nlargest: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.groupby.SeriesGroupBy.nsmallest: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.apply: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.transform: Sections are in the wrong order. Correct order is: Parameters, Returns, Examples
pandas.core.resample.Resampler.pipe: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.fillna: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, References, Examples
pandas.core.resample.Resampler.mean: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.io.formats.style.Styler: Sections are in the wrong order. Correct order is: Parameters, Attributes, See Also, Notes
pandas.io.formats.style.Styler.background_gradient: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, Notes
pandas.io.formats.style.Styler.to_excel: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.api.extensions.register_dataframe_accessor: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.api.extensions.register_series_accessor: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.api.extensions.register_index_accessor: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.api.extensions.ExtensionDtype: Sections are in the wrong order. Correct order is: See Also, Notes
pandas.Series.as_matrix: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.Series.divide: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.divmod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.from_csv: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also
pandas.Series.multiply: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.rdivmod: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Series.subtract: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.DataFrame.as_matrix: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.DataFrame.divide: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.multiply: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.subtract: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Panel.as_matrix: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.Panel.asof: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Panel.at_time: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Panel.between_time: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises, See Also, Examples
pandas.Panel.convert_objects: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also
pandas.Panel.empty: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Panel.fillna: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Panel.infer_objects: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.Panel.mask: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Panel.pipe: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes
pandas.Panel.slice_shift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.Panel.to_xarray: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Panel.tshift: Sections are in the wrong order. Correct order is: Parameters, Returns, Notes
pandas.Panel.where: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples

alexander-ponomaroff commented:
**REMAINING ERRORS**

```
pandas.Series.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Series.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.DataFrame.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.Timestamp.round: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises
pandas.Interval: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.Interval.overlaps: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.Timedelta.isoformat: Sections are in the wrong order. Correct order is: Returns, See Also, Notes, Examples
pandas.Timedelta.round: Sections are in the wrong order. Correct order is: Parameters, Returns, Raises
pandas.core.window.Rolling.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.window.Rolling.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.core.window.Expanding.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.window.Expanding.quantile: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Examples
pandas.core.groupby.GroupBy.head: Sections are in the wrong order. Correct order is: See Also, Examples
pandas.core.groupby.GroupBy.mean: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.core.groupby.GroupBy.nth: Sections are in the wrong order. Correct order is: Parameters, See Also, Examples
pandas.core.groupby.GroupBy.tail: Sections are in the wrong order. Correct order is: See Also, Examples
pandas.core.groupby.DataFrameGroupBy.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.apply: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas.core.resample.Resampler.mean: Sections are in the wrong order. Correct order is: Returns, See Also, Examples
pandas.api.extensions.register_dataframe_accessor: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.api.extensions.register_series_accessor: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
pandas.api.extensions.register_index_accessor: Sections are in the wrong order. Correct order is: Parameters, See Also, Notes, Examples
```

alexander-ponomaroff commented:
```
pandas/core/series.py - pandas.Series.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas/core/series.py - pandas.Series.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Example
pandas/core/frame.py  -  pandas.DataFrame.agg: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
pandas/core/frame.py  -  pandas.DataFrame.aggregate: Sections are in the wrong order. Correct order is: Parameters, Returns, See Also, Notes, Examples
```

Cannot be easily fixed - because it appends two different docs 
One from core/generic.py - _shared_docs['aggregate’] - parameters, returns, notes
One from core/series.py - _agg_doc - see also, examples

Some of the remaining 24 errors can still be fixed, I just spent a lot of time on this and need a break. But some of them like I stated above cannot be resolved because they append two different docs from different files and that affects the order.

@datapythonista

datapythonista reviewed (approved):
lgtm, thanks a lot for taking care of all those @alexander-ponomaroff

datapythonista commented:
@alexander-ponomaroff can you take a look at the linting errors in azure? seems like there are whitespaces in some lines you edited: https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=4807

Running locally `flake8 .` should report all the problems too, and should not return anything once they are fixed.

jreback commented:
@datapythonista it seems azure doesn't fail the build (the check here) when the linting job fails. can you fix this ASAP. IOW this PR should be an X, not in progress.

jreback commented:
@datapythonista on https://github.com/pandas-dev/pandas/pull/24126#issuecomment-444869658

thanks. pls keep us posted.

TomAugspurger commented:
Does this PR add linting so we don't regress these changes?

datapythonista commented:
We can't, as there are some tricky cases that will be addressed in a follow up. But I'll be adding to the validation all the errors that we fully fix, I'm on it.

TomAugspurger commented:
Can we at least do a whitelist, to ensure that the correct ones aren't
re-broken?

Some context: these types of changes are hard to keep around when working
on large changes that may involve moving
methods between files (like the DatetimeArray PR). I'd like to not lose all
this work.

On Thu, Dec 6, 2018 at 8:40 AM Marc Garcia <notifications@github.com> wrote:

> We can't, as there are some tricky cases that will be addressed in a
> follow up. But I'll be adding to the validation all the errors that we
> fully fix, I'm on it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/pandas-dev/pandas/pull/24126#issuecomment-444892616>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABQHIhicA2D29AIr_HHkdbQn_-2KpWamks5u2SxLgaJpZM4ZF-HU>
> .
>

datapythonista commented:
That makes sense, but we'll need to implement a `--ignore` option in the validation script. So far we can validate all docstrings that start with `pandas.Series.` or things like that. But for this PR, I expect to have a follow up with the rest of the fixes in the next couple of days. I'm on it now, creating issues for all the trivial formatting issues in the docstrings, so I wouldn't worry much, as people is helping a lot with them, and I expect to have all the easy ones fixed in the next couple of weeks.

For the rest (missing examples, parameters not documented...). It may make even more sense what you propose. But may be with what we have it's enough to progressively fix things and lint them (for example, focussing in `pandas.Series.str`, fix 100% of the docstrings there, and add validation for all the possible errors. But let's see once we've got all the formatting fixed, which will be very soon.

alexander-ponomaroff committed:
Fix: error W293: blank line contains whitespace

datapythonista commented:
@alexander-ponomaroff having huge PRs is not very efficient for the reviews. Every time you make some changes and get some feedback, we need to review the whole of it again. When I create issues I divide all these fixed in chunks, among other things because of this.

So, I prefer to merge this as soon as the CI is green, and address anything else in a separate PR.

alexander-ponomaroff commented:
@datapythonista I just fixed all remaining errors, with 13 errors that combine different docs left. I already did this on this branch. Could I merge this and link the files that I changed this time? I will keep this in mind and next time divide the fixes in a few sections to make it easier to review. Sorry for the inconvenience.

alexander-ponomaroff commented:
@datapythonista Actually nevermind, I don't want to create extra difficulties. Just merge this first and I will create a separate PR for the other ones.

jreback commented:
thanks, yeah the validation / final fixes are better done in a part 2
